latest_migrations.manifest

```diff
@@ -3,7 +3,7 @@ auth: 0012_alter_user_first_name_max_length
 axes: 0006_remove_accesslog_trusted
 contenttypes: 0002_remove_content_type_name
 ee: 0004_enterpriseeventdefinition_enterprisepropertydefinition
-posthog: 0156_insight_short_id
+posthog: 0158_auto_20210617_1952
```

Comment:
Since this hasn't been merged yet, you can convert it into just a single migration, instead of two: removing metric_name and adding metrics.

Otherwise, LGTM

Reply:
ah yes ofc - sorry about that, pushed too quickly